src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -736,6 +742,22 @@ public static bool IsSingleton(string set)
                 return false;
         }
 
+        /// <summary>
+        /// <c>true</c> if the set contains a single character only
+        /// </summary>
+        public static bool IsDoubleton(string set)
```

Comment:
Rather than having the caller do:
```C#
if (IsDoubleton(set))
{
    (char a, char b) = DoubletonChars(set);
    ...
}
```
how about:
```C#
if (IsDoubleton(set, out char a, out char b))
{
    ...
}
```
?

Reply:
Was following existing pattern, but since I'll hold this for 2.2, I'll clean it up.

Reply:
It's more that they appear to be a tight pair: you call DoubletonChars if and only if you've just called IsDoubleton, seems strange to me.

Reply:
ah my bad, I misread your initial comment. Right converging these two makes sense.